src/pip/_internal/operations/install/legacy.py

```diff
@@ -30,16 +29,15 @@ def write_installed_files_from_setuptools_record(
     root: Optional[str],
     req_description: str,
 ) -> None:
-    def prepend_root(path):
-        # type: (str) -> str
+    def prepend_root(path: str) -> str:
         if root is None or not os.path.isabs(path):
             return path
         else:
             return change_root(root, path)
 
     for line in record_lines:
         directory = os.path.dirname(line)
-        if directory.endswith('.egg-info'):
```

Comment:
Normally all this is excluded from black in [pre-commit config](https://github.com/pypa/pip/blob/87aee20df2890a89f70eecbd7086262146a2e64a/.pre-commit-config.yaml#L23) but it would have been better to exclude it as part of the black config directly in `pyproject.toml` (not in this PR). I'm not sure if it's worth reverting the black related changes here.

Reply:
Thanks for the catch, I think my code editor (which automatically runs black on save) did this.